### DISCUSSION TITLE
Wonder your conflict-solving workflow in nvim

### DISCUSSION THREAD

**Discussion by Saafo** [ayamir/nvimdots#823] [General]
I've noticed there is a plugin [diffview.nvim](https://github.com/sindrets/diffview.nvim) in the repo which can help us list all conflict files, and open each one in diff mode. I wonder how you solve conflicts? Do you solve it in neovim? Will you use `diffview.nvim` to solve or other plugins? What's your workflow to solve conflicts? Which layout do you use?

One of the most basic question I wonder is that how can I simply `use local` or `use remote` changes in hunks? I've tried `do` or `dg` in `diffview.nvim` but it didn't work. `:diffget 2` or `:diffget 3` neither works.

**Comment by ayamir** [comment]
Personally rarely used it. Just `lazygit` is fine :)

**Reply by Saafo** [reply]
Can lazygit solve conflicts? I mean, editing the file, I think merging tool must be an editor or something like that?

**Reply by ayamir** [reply]
I use lazygit just for some little diff change. Current working on Unity, so I mainly use `Rider` and `Unity Editor`.

**Reply by kang8** [reply]
> Can lazygit solve conflicts? I mean, editing the file

Can't editing the file in edit, but can solve conflicts in lazygit see [Resolving merge conflicts
](https://github.com/jesseduffield/lazygit#user-content-resolving-merge-conflicts)

**Reply by Saafo** [reply]
> But can solve conflicts in lazygit see [Resolving merge conflicts](https://github.com/jesseduffield/lazygit#user-content-resolving-merge-conflicts)

@kang8  Really thanks for the tip! I had never realized that lazygit can resolve simple confilcts! Really thanks again!!

**Comment by Jint-lzxy** [comment]
`:help diffview-merge-tool`. I personally prefer [vim-fugitive](https://github.com/tpope/vim-fugitive) b/c I've been using this back to the time when I started to use Vim :)
